pkg/util/azureclient/mgmt/network/securitygroups_addons.go

```diff
@@ -24,6 +25,14 @@ func (c *securityGroupsClient) CreateOrUpdateAndWait(ctx context.Context, resour
 	return future.WaitForCompletionRef(ctx, c.Client)
 }
 
+func (c *securityGroupsClient) DeleteAndWait(ctx context.Context, resourceGroupName string, networkSecurityGroupName string) (err error) {
+	future, err := c.Delete(ctx, resourceGroupName, networkSecurityGroupName)
```

Comment:
In the light of https://github.com/Azure/ARO-RP/pull/1679 would you please reference the original client on this line? 
```suggestion
	future, err := c.SecurityGroupsClient.Delete(ctx, resourceGroupName, networkSecurityGroupName)
```